### ISSUE TITLE
Support Travis folding

### ISSUE THREAD

**Issue by IDragonfire** [mendhak/Gradle-Travis-Colored-Output#1]
Travis support folding of tests in the log ...

# Example 
```java
tasks.withType(Test) {
    beforeSuite { suite ->
        out.println("travis_fold:start:" + suite.name);
        out.println(suite.name);
    }
    afterSuite { suite ->
        out.println("travis_fold:end:" + suite.name);
    }
}
```

It is possible to integrate?

**Comment by mendhak** [comment]
That's a really nice feature you found.  I've now added some lines in to do suite folding.  Can you give it a try see if it works alright.

I made a few attempts:  https://travis-ci.org/mendhak/gpslogger/builds/231903734

And also with a failing test:  https://travis-ci.org/mendhak/gpslogger/builds/231902014 

I've also added a variable if anyone wants to enable folding:

    apply from: 'buildtools/ColoredOutput.gradle'
    project.ext.set("TRAVIS_FOLDING", true)

I say 'anyone' but as far as I know you're probably the only user :grin: 

Edit - changed - folding is disabled by default.

**Comment by IDragonfire** [comment]
I currently used this here https://github.com/FAForever/faf-java-api/pull/89/commits/9618a65276abae8007965687de4b30f5afa035de
but your implementation looks better ;)

**Comment by IDragonfire** [comment]
@mendhak are are you very experienced with gradle?
We need the `--info` parameter but I found no possible to fold the log.
e.g. for the dependency management ...
https://travis-ci.org/FAForever/faf-java-api/builds/231914408

I tried to create tasks for `compileJAVA`, but without any chance ...
Looks like the INFO logging is outside of the tasks ...

**Comment by mendhak** [comment]
Yeah the afterSuite/beforeSuite get called as close to the tests as possible so no chance of including it in the generated output. None of Gradle's own logging reaches the script.

But no I'm not very gradle experienced, I usually forget how things are done once I get it done :D 

In my case after I learned about travis folding from you I changed my [travis](https://github.com/mendhak/gpslogger/blob/travisfold/.travis.yml#L25-L40) to do the building in the before_script then ran just the tests in the script section.  Not sure how far that will help, or if your project is OK with not using `--info` in some places
